api/cluster/templater.go

```diff
@@ -48,11 +48,9 @@ const (
 	envPredictorStorageURI       = "STORAGE_URI"
 	envPredictorDisableLiveness  = "MERLIN_DISABLE_LIVENESS_PROBE"
 
-	defaultPredictorArtifactLocation  = "/mnt/models"
-	defaultPredictorPort              = "8080"
-	defaultTransformerPort            = "8080"
-	defaultPredictorDisableLiveness   = "false"
-	defaultTransformerDisableLiveness = "false"
```

Comment:
Why did these lines got deleted? Is it github rendering issue or actual deletion?

Reply:
Not used anymore, forgot to delete in last PR